src/styles/blocks/catalog.scss

```
@@ -0,0 +1,20 @@
+@import "./card.scss";
+@import "./star.scss";
+
+.cards {
+  padding: 50px 0;
+  max-width: 944px;
+  margin: 0 auto;
+
+  &__row {
+    display: flex;
+
+    > :not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
+  }
+
+  > :not(:last-child) {
+    margin-bottom: 46px;
+  }
```

```suggestion
  &:not(:last-child) {
    margin-bottom: 46px;
  }
```

you are applying this rule to the `row`, not to its children 😉